spec/domain/reports/series_spec.rb

```diff
@@ -129,4 +129,24 @@
       expect(described_class.new.content_items).to match_array([item1, item2])
     end
   end
+
+  describe '#include_edition_metrics' do
```

Comment:
The method is private so we shouldn't test it.
I would create a fixture of data so the code need to check if the metrics are editions or not, then run the code, and check each option.